core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -377,17 +373,10 @@ oppia.controller('StateResponses', [
             };
             $scope.tmpRule = RuleObjectFactory.createNew(null, {});
             $scope.tmpOutcome = OutcomeObjectFactory.createNew(
-              EditorStateService.getActiveStateName(), [''], []);
+              EditorStateService.getActiveStateName(), '', []);
 
             $scope.isSelfLoopWithNoFeedback = function(tmpOutcome) {
-              var hasFeedback = false;
-              for (var i = 0; i < tmpOutcome.feedback.length; i++) {
-                if (tmpOutcome.feedback[i]) {
-                  hasFeedback = true;
-                  break;
-                }
-              }
-
+              var hasFeedback = tmpOutcome.feedback != null;
```

Comment:
Ditto, here and below.

Btw, also, please use !== rather than != in general.

Reply:
done